about.html

```diff
@@ -0,0 +1,145 @@
+                    </div>
+                    <div id="menu-link-cont">
+                    <a class="menu-link">About</a>
+                    <a class="menu-link">Program</a>
+                    <a class="menu-link">Speakers</a>
+                    <a class="menu-link">Join</a>
+                    </div>
+                </div>
+
+
+
+            </nav>
+
+
+            <!-- Mobile Navigation Menu (Hidden for Desktop Version) -->
+            <nav id="mobile-navbar" class="navbar-menu mobile-navbar">
+                <a id="hamburger-link" href="#" class="toolbar-button"><img id="hamburger" class="hamburger" src="images/icons/mobile-hamburger-icon.png" alt="Open Menu"></a>
+            </nav>
+
+        </div>
+
+       
+        <div id="heading-cont" class="heading">
+            <h2 id="main-subtitle" class="about-subtitle">International Culinary Summit</h2>
+            <h1 id="main-title" class="about-title">FLAVORS OF PERU 2022</h1>
+        </div>
+
+        <div id="about-description-cont">
+            <p class="about-description">The international culinary summit laureated chefs, bartenders, sommeliers and other gastronomy experts to join the celebration of the senses. One of Perus most sacred treasures is memoired in this exclusive and once-in-a-lifetime experience.'</p>
+        </div>
+
+        <div id="about-contact">
+            <p class="about-description">Please contact us per Email for any further questions about FLAVORS OF PERU 2022.</p>
+            <a href="#" id="about-email">contact@flavorsofperu22.com</a>
+        </div>
+
+    </header>
+
+    <section id="about-logo">
+
+        <h2 id="about-logo-info" class="about-section-title">The FLAVORS OF PERU summit logo</h2>
+
+        <hr class="about-hr">
+
+        <p class="about-text-content">The logo of FLAVORS OF PERU 2022 is property of MARCA PERU and the office of tourism and foreign commerce of the Ministry of Tourism. </p>
+
+
+        <div class="about-logo-container">
+
+            <img src="images/feature-images/event-logo-black.png" class="event-logo" alt="Flavors of Peru logo">
+
+        </div>
+
+    </section>
+
+    <section id="about-past-events">
+
+        <h2 id="about-join-info" class="about-section-title">See the past Flavors of Peru editions</h2>  
+
+        <hr class="about-hr hr-past-event">
+
+        <p class="about-text-content">Take a look at the last two Flavors of Peru summits which took place in Moscow and Lisbon.</p>
+
+
+        <div id="events-container">
+            
+            <div class="about-img-events-Lisbon">    
+                <h2 class="event-img-year inside-text">2019</h2>
+                <h3 class="event-img-location inside-text">Flavors of Peru Lisbon</h3>
+            </div>
+
+            
+            <div class="about-img-events-Moscow">    
+                <h2 class="first-text">2018</h2>
+                <h3 class="second-text">Flavors of Peru Moscow</h3>
+            </div>
+        </div>
```

Comment:
- [ ] Please, work on these images and text to be more centered, at the same time, try to not cut some partners. 🐦 
![image](https://user-images.githubusercontent.com/20563252/137224717-670b10a8-a3d4-4771-bb87-e07712656c8d.png)